components/service-operator/README.md

```diff
@@ -1,16 +1,49 @@
 # Service Operator
 
+We aim for this to be our generic service operator, with a hope for it to be a
+provider agnostic one. The reason being, a desire to run the stack locally,
+without the worry of providing live database connections to a local deployment.
+
+## Adding new services
+
+Say you have a desire to add a new service to the mix. We're using
```

Comment:
Can we have an ADR about this?

I think there's an interesting and nontrivial decision about new services:

 - we have one operator that handles them all
 - we have separate operators for each (eg postgres-operator, sqs-operator, etc)

My uninformed prejudice is towards the second, so if we're going with the first can we write down why?

Reply:
See next commit.